ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -203,7 +204,8 @@ public override void OnDropViewInstance(ThemedReactContext reactContext, Border
         {
             base.OnDropViewInstance(reactContext, view);
 
-            _imageSources.Remove(view.GetTag());
+            List<KeyValuePair<string, double>> sources;
+            _imageSources.TryRemove(view.GetTag(), out sources);
```

Comment:
>_imageSources.TryRemove(view.GetTag(), out sources); [](start = 12, length = 52)

I think with new C# syntax you can just write:
```
_imageSources.TryRemove(view.GetTag(), out List<...> unused);
```

Reply:
or `_imageSources.TryRemove(view.GetTag(), out _);`